src/Testing/ZopeTestCase/zopedoctest/testFunctionalDocTest.py

```diff
@@ -89,23 +89,24 @@ def test___str___w_headers(self):
 
 
 SHOW_COOKIES_DTML = '''\
-<dtml-in "REQUEST.cookies.keys()">
+<dtml-in "list(REQUEST.cookies.keys())">
```

Comment:
This is admittedly me papering over an issue, so I can see the other errors.

This failed with something like SafeIter has no attribute `__len__` and that's most likely an AccessControl issue.